Sources/SwiftGodot/Variant.swift

```diff
@@ -273,7 +273,8 @@ public class Variant: Hashable, Equatable, CustomDebugStringConvertible {
             if valid == 0 || oob != 0 {
                 return nil
             }
-            return Variant(copying: _result)
+                        
+            return Variant(takingOver: _result)
```

Comment:
Should probably add a comment here. Unlike `Array`, that gives a pointer to its internal storage, this API returns a copy.